index.html

```diff
@@ -284,21 +271,40 @@
 
   // This is creating an event listener, a function that the browser will call every time the specified
   // event is "fired" by the selected element (here the button with the id 'sort-asc').
+
+  // Sort Category Event Listeners
+  document.getElementById('sort-clinton').addEventListener('click', function () {
+    if (options.candidate === 'clinton') {
+      options.candidate = false;
+    } else {
+      options.candidate = 'clinton';
+      options.sorted = false;
```

Comment:
This is an interesting two-step process that you're setting up. It's pretty difficult for the user to understand the logic behind it. Consider:

Click "Clinton" > Click "Ascending" -> Sorts in ascending order of Clinton delegates
Click "Ascending" > Click "Clinton" -> I'd expect this to work the same as the above, but it instead does nothing (because the above sets `options.sorted` to `false`)
Click "Clinton" > Click "Ascending" > Click "Descending" -> Sorts in descending order of Clinton delegates
Click "Clinton" > Click "Ascending" > Click "Sanders" -> I'd expect this to sort in ascending order of Sanders delegates, but it instead does nothing

You're trying to force the user to click first on the candidate and then on the sort direction, but there's nothing that communicates that. And I'd suggest that's probably never what we want to force.

Simple approach: Exactly what you have, but avoid the interactions between the two that come from the `options.sorted = false` statements in the candidate event listeners.

Advanced approach: It doesn't make sense for someone to be able to click a sort order if they haven't selected a candidate. So those buttons should be disabled by default (Alex showed how to set this in D3 with `d3.selectAll(...).attr('disabled', true/false`). When a candidate is selected, those order buttons should become enabled (and it probably makes sense to select descending by default, so that the chart immediately changes). And if I deselect a candidate, I should disable the order buttons. (What should happen if descending is selected, and I click again on descending? It probably doesn't make sense for me to ever be able to have a candidate selected by no order selected...)